### PR TITLE
Create nalia_dearnise.txt

### LINKED ISSUE #906: Nalia de'Arnise

### PR DESCRIPTION
**Pull request by squee1968**
-closes #906

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/nalia_dearnise.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+ManaCost:1 W B
+Types:Legendary Creature Human Rogue
+PT:3/3
+S:Mode$ Continuous | Affected$ Card.TopLibrary+YouCtrl | AffectedZone$ Library | MayLookAt$ You | Description$ You may look at the top card of your library any time.
+S:Mode$ Continuous | Affected$ Cleric.TopLibrary+YouCtrl+nonLand,Rogue.TopLibrary+YouCtrl+nonLand,Warrior.TopLibrary+YouCtrl+nonLand,Wizard.TopLibrary+YouCtrl+nonLand | AffectedZone$ Library | MayPlay$ True | Description$ You may cast Cleric, Rogue, Warrior, and Wizard spells from the top of your library.
+T:Mode$ Phase | Phase$ BeginCombat | ValidPlayer$ You | TriggerZones$ Battlefield | CheckSVar$ X | SVarCompare$ EQ4 | Execute$ PutCounterAll | TriggerDescription$ At the beginning of combat on your turn, if you have a full party, put a +1/+1 counter on each creature you control and those creatures gain deathtouch until end of turn.
+SVar:PutCounterAll:DB$ PutCounterAll | ValidCards$ Creature.YouCtrl | CounterType$ P1P1 | CounterNum$ 1 | SubAbility$ DBPumpAll
+SVar:DBPumpAll:DB$ PumpAll | ValidCards$ Creature.YouCtrl | KW$ Deathtouch
+SVar:X:Count$Party
+DeckHas:Ability$Party
```
Comment (Agetian) [inline forge-gui/res/cardsfolder/upcoming/nalia_dearnise.txt:11]: DeckHas:Ability$Party|Counters

File: forge-gui/res/cardsfolder/upcoming/nalia_dearnise.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Name:Nalia de'Arnise
+ManaCost:1 W B
+Types:Legendary Creature Human Rogue
+PT:3/3
+S:Mode$ Continuous | Affected$ Card.TopLibrary+YouCtrl | AffectedZone$ Library | MayLookAt$ You | Description$ You may look at the top card of your library any time.
+S:Mode$ Continuous | Affected$ Cleric.TopLibrary+YouCtrl+nonLand,Rogue.TopLibrary+YouCtrl+nonLand,Warrior.TopLibrary+YouCtrl+nonLand,Wizard.TopLibrary+YouCtrl+nonLand | AffectedZone$ Library | MayPlay$ True | Description$ You may cast Cleric, Rogue, Warrior, and Wizard spells from the top of your library.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/nalia_dearnise.txt:6]: I think `YouOwn` is better than `YouCtrl` for all these

File: forge-gui/res/cardsfolder/upcoming/nalia_dearnise.txt [hunk 3]
```diff
@@ -0,0 +1,14 @@
+Name:Nalia de'Arnise
+ManaCost:1 W B
+Types:Legendary Creature Human Rogue
+PT:3/3
+S:Mode$ Continuous | Affected$ Card.TopLibrary+YouCtrl | AffectedZone$ Library | MayLookAt$ You | Description$ You may look at the top card of your library any time.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/nalia_dearnise.txt:5]: ah YouOwn here too

File: forge-gui/res/cardsfolder/upcoming/nalia_dearnise.txt [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+S:Mode$ Continuous | Affected$ Cleric.TopLibrary+YouCtrl+nonLand,Rogue.TopLibrary+YouCtrl+nonLand,Warrior.TopLibrary+YouCtrl+nonLand,Wizard.TopLibrary+YouCtrl+nonLand | AffectedZone$ Library | MayPlay$ True | Description$ You may cast Cleric, Rogue, Warrior, and Wizard spells from the top of your library.
+T:Mode$ Phase | Phase$ BeginCombat | ValidPlayer$ You | TriggerZones$ Battlefield | CheckSVar$ X | SVarCompare$ EQ4 | Execute$ PutCounterAll | TriggerDescription$ At the beginning of combat on your turn, if you have a full party, put a +1/+1 counter on each creature you control and those creatures gain deathtouch until end of turn.
+SVar:PutCounterAll:DB$ PutCounterAll | ValidCards$ Creature.YouCtrl | CounterType$ P1P1 | CounterNum$ 1 | SubAbility$ DBPumpAll
+SVar:DBPumpAll:DB$ PumpAll | ValidCards$ Creature.YouCtrl | KW$ Deathtouch
+SVar:X:Count$Party
+DeckHas:Ability$Party
+DeckHints:Type$Cleric|Warrior|Wizard
+AI:RemoveDeck:Random
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/nalia_dearnise.txt:13]: Why the RemoveDeck here?

**Comment by Agetian** [comment]
Looks like this PR can be merged after a few simple tweaks. Is the original submitter available to make the necessary changes?

**Comment by squee1968** [comment]
> Looks like this PR can be merged after a few simple tweaks. Is the original submitter available to make the necessary changes?

Sorry that I kinda turned into a cookie and was a wafer awhile.

**Review by Agetian — APPROVED** [review]

**Pull request merged**